### DISCUSSION TITLE
[CODE ADVICE] : Trying to tinker the code for counting dbin packages.

### DISCUSSION THREAD

**Discussion by lidgnulinux** [fastfetch-cli/fastfetch#1430] [Q&A] [answered]
Hello I'm trying to add `dbin` package detection. It works but the code seems not general, at `src/detection/packages/packages_linux.c` I  add : 

```
if (!(options->disabled & FF_PACKAGES_FLAG_DBIN_BIT)) packageCounts->dbin += getNumElements(baseDir, "/home/my_user/.local/dbin", DT_REG);
```

How to make `/home/my_user/.local/dbin` act like `~/.local/dbin` ? When I change `/home/my_user/.local/dbin` to `~/.local/dbin`, it won't show amount of my dbin packages.

Any clue ? Thanks.

**Comment by CarterLi** [comment, marked as answer]
Please refer to https://github.com/fastfetch-cli/fastfetch/blob/dev/src/detection/packages/packages_linux.c#L643-L680

**Reply by lidgnulinux** [reply]
Thanks.
